docs_src/general/EdgeX_CN.md

```diff
@@ -0,0 +1,153 @@
+
+!!! Note
+    There are some edge use case where a sensor-to-cloud architecture is warranted.  Where the sensor speaks well known IT protocols (TCP/IP REST, MQTT, etc.), the edge data collection rates are small, and there is no need to make quick decisions at the edge, a simple sensor to cloud architecture makes sense and would likely negate the need for EdgeX in that situation.
+
+### Other CN Aspects
+Here are some  other aspects or thoughts associated to the CN approach (directly or by loose association) and how they apply to EdgeX. 
+
+**OS is separate**
+
+As highly abstracted, containerized applications, cloud native apps do not have a dependency on any specific operating system or individual machine.  EdgeX is, for the most part, platform agnostic and able to run on any hardware, OS or connect to any type of sensor or cloud system (whether using EdgeX containers or running on bare metal).  However, there are some sensors/devices that require OS or hardware specific drivers or protocol support.  These specific services (typically device services) are OS dependent.
+
+**High Availability**
+
+While not strictly a CN principal, CN container apps are typically said to provide high availability (HA) - avoiding downtime (scheduled or unscheduled), often by taking advantage of CN infrastructure like Kubernetes to keep multiple instances of a service running when HA is paramount.  EdgeX does not offer HA out of the box.  Services are built to be resilient (for example, recovering from anticipated errors or waiting for dependent services to come up or return when they are not detected), but they are not guaranteed to be HA.  When EdgeX services are run in some environments (snaps for example) the environment may detect service issues and launch a new instance of the service to prevent downtime, but these are features of the underlying runtime environments and not of EdgeX services directly.  HA often requires a certain amount of redundancy; that is keeping multiple instances of a service running (or at the ready) and using something like Kubernetes to route traffic appropriately given the condition of a service.  EdgeX does not have this infrastructure built in, and even if it did, it would have difficulty since some services are again tied to physical senors/devices.  If a device service connected to a Modbus device, for example, was to go down, then a backup/redundant service would be of little use without re-provisioning the sensor or device to the backup device service.  In order to provide true HA uptime with an edge solution that includes EdgeX, one would need to scale out not up.  That is, one would need to setup redundant hardware (sensors, gateway, etc.) with the edge application (EdgeX in this instance) connected to its copy of the sensors and devices and each transmitting back to the IT enterprise such that the enterprise could compare and detect when one of the copies was likely having issues.
```

Comment:
I think simpler arguments can be made here:
- We don't have any use cases (that I know of) where customers are hooking up multiple pressure transducers, multiple valves, etc in an attempt to build in redundancy.  Maybe an exception is my RV water heater, that needs both solenoids to open before the propane will flow.
- We don't have any use cases (that I know of) where the devices are active components in the system and we are worried about missing data due to inability to capture and process it
- It is not cost-effective to manually code HA; the HA requirement would necessarily require building on a compute infrastructure like Kubernetes that is designed for HA, and eschewing support for native processes, snaps, and Docker that weren't designed HA.

Reply:
@bnevis-i - I think you have provided good arguments for why EdgeX is not pursuing these.  I think my explanation provides the current state and talks about how it works today (which is not HA).  I like your list of "arguments" for why EdgeX won't attempt to be HA in the near future and so added them to this part of the doc.  I am